pyravendb/raven_operations/timeseries_operations.py

```diff
@@ -0,0 +1,161 @@
+from typing import Optional, Iterable, List, Generator
+from pyravendb.commands.raven_commands import *
+from pyravendb.tools.utils import Utils
+from .operations import Operation
+from datetime import datetime
+
+import sys
+
+
+class TimeSeriesRange:
+    def __init__(self, name, from_date: Optional[datetime] = None, to_date: Optional[datetime] = None):
+        if not name:
+            raise ValueError("TimeSeriesRange must include name")
+        self.name = name
+        self.from_date = from_date
+        self.to_date = to_date
```

Comment:
This should default to `9999-12-31` when set to `None`

Reply:
same